gradle/verification-metadata.xml

```diff
@@ -1275,6 +1275,11 @@
             <sha256 value="aa372691b684705b04178a43984bebe58e08b3d9a4c90e868b33d873646cba9a" origin="Artifact is not signed" reason="Artifact is not signed"/>
```

Comment:
I believe that entry can be removed.

Reply:
Not yet because the wrapper still uses 3.2.2. That entry will be removed in a subsequent PR updating the wrapper.